CopyOfBreakingChange.md

```diff
@@ -8,6 +8,10 @@ Bracketed text is a placeholder; replace the text with the requested information
 
 [Brief description of the change]
 
+### Preview Version
+
+[Preview Version in which the breaking change first occurred (for example, 6.xx)]
```

Comment:
We don't need a new section. We just need to add the preview version to the next section, Version introduced.